drivers/can/Kconfig.rcar

```
@@ -11,6 +11,7 @@ config CAN_RCAR
 
```

Can we do with a if statement, it looks more logical to me, and will be coherent with mcp2515 driver ?

```suggestion
if CAN_RCAR

```

---

Sure.  But I thought we use `depends on` for single symbol.  (aka. https://github.com/zephyrproject-rtos/zephyr/commit/eddd98f8118709aa29c0f14785749199be224a41 and many other commits related to it)

---

indeed miss that one. So yes better use depends on